redfish-core/lib/fabric_adapters.hpp

```diff
@@ -221,14 +303,27 @@ inline void doAdapterGet(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
     aResp->res.jsonValue["Status"]["State"] = "Enabled";
     aResp->res.jsonValue["Status"]["Health"] = "OK";
 
+    // add pcieslots
+    getFabricAdapterPCIeSlots(
+        aResp, systemName, fabricAdapterPath,
+        [aResp](const std::string& chassisName,
+                const dbus::utility::MapperEndPoints& /*pcieSlotPaths*/) {
+        aResp->res.jsonValue["Oem"]["@odata.type"] = "#OemFabricAdapter.Oem";
+        aResp->res.jsonValue["Oem"]["IBM"]["@odata.type"] =
+            "#OemFabricAdapter.IBM";
+        aResp->res.jsonValue["Oem"]["IBM"]["Slots"]["@odata.id"] =
+            crow::utility::urlFromPieces("redfish", "v1", "Chassis",
+                                         chassisName, "PCIeSlots");
+        });
+
     getFabricAdapterLocation(aResp, serviceName, fabricAdapterPath);
     getFabricAdapterAsset(aResp, serviceName, fabricAdapterPath);
     getFabricAdapterState(aResp, serviceName, fabricAdapterPath);
     getFabricAdapterHealth(aResp, serviceName, fabricAdapterPath);
     getLocationIndicatorActive(aResp, fabricAdapterPath);
 
-    // if the adapter also implements this interface, link the adapter schema to
-    // PCIeDevice schema for this adapter.
+    // if the adapter also implements this interface, link the adapter
+    // schema to PCIeDevice schema for this adapter.
```

Comment:
no sure why this needed to change? Oh well

Reply:
This is changed automatically by 'clang-format-16'.